karma/test-controllers.coffee

```diff
@@ -541,6 +543,21 @@ describe('controllers:', () ->
 
         expect(MessageService.bulkFlag).toHaveBeenCalledWith([test.msg2], true)
       )
+
+      it('onCaseWithoutMessage', () ->
+        createCaseModal = spyOnPromise($q, $scope, ModalService, 'create_case')
+        openCase = spyOnPromise($q, $scope, CaseService, 'open')
+        redirect = spyOnPromise($q, $scope, UtilsService, 'navigate')
+
+        $scope.onCaseWithoutMessage()
+        expect(ModalService.create_case).toHaveBeenCalledWith({title: 'Create case'})
+
+        createCaseModal.resolve({text: 'test summary', partner: 2, user: 3, urn: 'tel:123'})
+        expect(CaseService.open).toHaveBeenCalledWith(null, 'test summary', 2, 3, 'tel:123')
+
+        openCase.resolve({is_new: false, id: 7})
+        expect(UtilsService.navigate).toHaveBeenCalledWith('case/read/7/?alert=open_found_existing')
```

Comment:
Resolved in https://github.com/praekelt/casepro/pull/109/commits/bddff6d36e978e580de5752a572a4967abd9e191